commercetools-models/src/test/java/io/sphere/sdk/carts/ExternalTaxRatesIntegrationTest.java

```diff
@@ -240,7 +241,7 @@ public void addCustomShippingMethod() {
             final double taxRate = 0.20;
             final ExternalTaxRateDraft externalTaxRate =
                     ExternalTaxRateDraftBuilder.ofAmount(taxRate, taxRateName, DE).build();
-            final ShippingRate shippingRate = ShippingRate.of(EURO_10);
+            final ShippingRate shippingRate = ShippingRate.of(EURO_10,null, Collections.emptyList());
```

Comment:
Here we should change our update action to use a draft instead.

Reply:
this seems to be rather a large subject so i created a separate ticket #1567

Reply:
Ok and great that you created a follow up 👍